docs/recipes/lighthouse-plugin-example/audits/preload-as.js

```diff
@@ -37,7 +37,7 @@ class PreloadAsAudit extends Audit {
     const passed = noAsLinks.length === 0;
 
     return {
-      rawValue: passed,
+      score: passed ? 1 : 0,
```

Comment:
`+passed`? idk how hacky you'll allow js to be :)

Reply:
I like the explicitness of this for the recipe :)